linchpin/shell/__init__.py

```diff
@@ -91,10 +90,12 @@ def _handle_results(ctx, results, return_code):
 
     # PRINT OUTPUT RESULTS HERE
     ctx.log_state(output)
+    use_actual_rcs = ast.literal_eval(lpcli.get_cfg('lp',
+                                                    'use_actual_rcs'))
 
```

Comment:
as a rule, you should add the .title() function to the item to get the 'True' or 'False' state before it's evaluated by the ast.literal_eval method. Something like this would be appropriate.

```
uar = lpcli.get_cfg('lp', 'use_actual_rcs'))
use_actual_rcs = ast.literal_eval(uar.title())
```
In this way, if the value is 'true', or 'false', or any combination thereof, the value will get converted to proper title case, and return the proper boolean.

Reply:
Okay , I was not aware of the rule . I tried ast.literal_eval(lpcli.get_cfg('lp','use_actual_rcs')) and it worked . Will make the change.